packages/utils/tsconfig.lib.json

```diff
@@ -5,13 +5,12 @@
     "declaration": true,
     "types": ["node"]
   },
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*.ts", "perf/**/*.ts"],
```

Comment:
shouldn't this be the other way round? Not included in the lib bundle and lib included in perf bundle?

Reply:
Added `tsconfig.perf.json`